src/App.css

```diff
@@ -1,15 +1,16 @@
 .app {
-    background-color: white;
+  background-color: white;
 } 
 
-.overlay {
-    position: relative;
+.app .editor-overlay {
+  width: 100%;
+  height: 100%;
 }
 
-.Pane1 {
-    background-color: rgba(211, 209, 209, 0.301);
+.app .Pane1 {
```

Comment:
You still have caps for class names :(

Reply:
Those are split panes name ... they come like that .... I don’t know how to change those

Reply:
Oh, I see. I believe the `Pane` components should accept `className`, so try that and supply your own.

Reply:
According to the docs for [react-split-pane](https://github.com/tomkp/react-split-pane#props) there is no `className` prop. I tried supplying my own but that is not supplied down to children. As we are using a `div` is considered as a pane and they were already names, I just used that name.

Reply:
If you see the prop-types here, it looks like it definitely takes className, but there’s also a couple of className props: https://github.com/tomkp/react-split-pane/blob/master/src/SplitPane.js